code/datums/emergency_calls/sons_of_mars_breachers.dm

```diff
@@ -0,0 +1,59 @@
+//Sons of Mars
+/datum/emergency_call/som_breachers
+	name = "Sons of Mars Squad"
+	base_probability = 13
+	alignement_factor = 0
+	///number of available special weapon dudes
+	var/max_specialists = 1
+
+/datum/emergency_call/som_breachers/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You [pick("grew up in the mines working in horrible conditions until signing up to", "were part of a poor family until you decided to leave to join", "were born on a colony on Mars to a family of several brothers and sisters before leaving to", "worked at horrible conditions in the mines until deciding to leave to join", "have a proud family tradition of military service so enrolled as early as possible to serve")] the Sons of Mars (SoM).</b>")
```

Comment:
`were born on a colony on Mars`

The SoM haven't lived on mars in centuries. This should be Cydonia, their Martian analogue & base of operations.

Reply:
Mars is still populated, not everyone left after the rebellion. I always assumed that line meant the SOM continued to actively recruit from the repressed people of Mars itself down the years.

Reply:
I thought this too but then I kind of realized it wouldn't make sense as Mars is a TGMC controlled world and there'd be loads of anti SoM propaganda to prevent that very thing.

Meaning that- for the average martian, the SoM are seen as violent fringe extremists.